styles/header.css

```diff
@@ -1,7 +1,16 @@
 :root {
-  --sul-logo-color: white;
+  --sul-logo-color: var(--stanford-black);
   --identity-bar-color: var(--stanford-cardinal);
 }
+
+.bg-dark {
+  --sul-logo-color: var(--bs-navbar-color);
+}
+
+.identity-bar {
+  --bs-dark-rgb: var(--stanford-cardinal-rgb);
```

Comment:
Isn't it odd that `--identity-bar-color` and `--bs-dark-rgb` are both cardinal? Shouldn't one be white?

Involving `.bg-dark` in this top bar seems confusing to me.

Reply:
do you think it'd be better to just add a class like `.bg-cardinal` and then apply that? I think I've seen that on some of our apps already.